### ISSUE TITLE
indentation

### ISSUE THREAD

**Issue by ink1** [krasserm/super-resolution#41]
https://github.com/krasserm/super-resolution/blob/59e91633378b75e515c774bb02e93300967c1a11/train.py#L83
should this line be one more level indented as tape exists in with context?

**Comment by krasserm** [comment]
You only need to put the operations into the `tape` context that need to be recorded. Computation of gradients using `tape` can be done outside this context. Please read the [GradientTape](https://www.tensorflow.org/api_docs/python/tf/GradientTape) documentation for details.
